spec.html

```diff
@@ -31069,7 +30933,7 @@
                   1. Let _nextValue_ be ? IteratorValue(_next_).
                   1. Append _nextValue_ to the end of the List _values_.
               1. Let _len_ be the number of elements in _values_.
-              1. Let _targetObj_ be ? AllocateTypedArray(_C_, _len_).
+              1. Let _targetObj_ be ? Construct(_C_, «_len_»).
```

Comment:
Lists should have spaces inside them (apply this to the rest of the patch too).

(Also we should probably update all instances of &raquo; and &laquo; with the proper character now that encoding is utf8).